DependencyInjection/Configuration.php

```diff
@@ -19,7 +19,12 @@ class Configuration implements ConfigurationInterface
     public function getConfigTreeBuilder()
     {
         $treeBuilder = new TreeBuilder('leogout_seo');
-        $rootNode = $treeBuilder->getRootNode();
+        // Keep compatibility with symfony/config < 4.2
+        if (\method_exists($treeBuilder, 'getRootNode')) {
```

Comment:
Version check can be better approach instead of method checking. You can use `Kernel::VERSION_ID` constant for this.